runtime/runtime/src/lib.rs

```diff
@@ -2150,6 +2217,7 @@ fn missing_chunk_apply_result(
     });
 }
 
+#[inline(never)]
```

Comment:
Just out of curiosity, why never inline? If it's non-trivial can you add a comment?

Reply:
It is trivial, I've been using it to get more frames into the profiler, it doesn't need to stay.

Reply:
Removed.